scripts/ci/format-lintro-pr-comment.py

```
@@ -0,0 +1,446 @@
+
+
+def _build_excerpt(lines: list[str]) -> str:
+    """Build a short diagnostic excerpt from normalized logs.
+
+    Args:
+        lines: Prefix-stripped job-log lines.
+
+    Returns:
+        Short excerpt suitable for a PR comment code block.
+    """
+    last_tool, last_tool_index = _find_last_tool(lines)
```

Ruff (0.15.10)

[warning] Unpacked variable `last_tool` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)